### ISSUE TITLE
Not able to edit alt text after posting

### ISSUE THREAD

**Issue by naivchan** [dahlia/hollo#538]
I had an image with Alt Text in a post that I wanted to edit, but I was unable to save my edits to the alt text from either phanpy or moshidon. 

When I click "save", phanpy states that the post edit was successful, but my new alt text does not appear. 

I'm currently on version 0.9.2 if that changes anything.

**Comment by dahlia** [comment]
Fixed in v0.8.8 and v0.9.6.

**Comment by naivchan** [comment]
> Fixed in v0.8.8 and v0.9.6.

Thank you! I was able to edit my alt text now
